lib/api/bucketPutVersioning.js

```diff
@@ -135,11 +136,15 @@ function bucketPutVersioning(authInfo, request, log, callback) {
         if (err) {
             log.trace('error processing request', { error: err,
                 method: 'bucketPutVersioning' });
+            monitoring.promMetrics(
+                'PUT', bucketName, err.code, 'bucketPutVersioning');
         } else {
             pushMetric('putBucketVersioning', log, {
                 authInfo,
                 bucket: bucketName,
             });
+            monitoring.promMetrics(
+                'PUT', bucketName, '200', 'putBucketVersioning');
```

Comment:
bucketPutVersioning or putBucketVersioning?

Reply:
same comment for bucketPutWebsite

Reply:
Because putshMetrics also uses a naming convention where the type of request like PUT, DELETE, HEAD, etc. comes first I used the same for metrics.